modules/tinymce/src/core/main/ts/bookmark/ResolveBookmark.ts

```diff
@@ -224,18 +230,18 @@ const resolveId = (dom: DOMUtils, bookmark: IdBookmark): Optional<Range> => {
       const rng = dom.createRng();
```

Comment:
Should we call this and other variables throughout this file `range` instead to match the resolve result property name (or change the result type to use `rng`)? That way we can shorten it to just `{ range, forward: isForwardBookmark(bookmark) }`.